### PR TITLE
[21.04] Fix and integrate the new Dockerimage stucture into 21.04

### PR DESCRIPTION
**Pull request by y0urself**
**What**:

**Why**:

**How**:

**Checklist**:

- [ ] Tests
- [ ] [CHANGELOG](https://github.com/greenbone/gvm-libs/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

File: .docker/testing/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    make \
+    pkg-config \
+    uuid-dev \
+    libssl-dev \
+    lcov \
+    libical-dev \
+    libpq-dev \
+    libnet1-dev \
+    postgresql-server-dev-all \
+    xsltproc \
```
Comment (bjoernricks) [inline .docker/testing/Dockerfile:44]: i am not sure about the dependencies. looks like the dependencies from gvmd to me. i don't think gvm-libs needs postgres at all and also not xsltproc. but maybe i am wrong.

File: .docker/testing/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+# & install cgreen for unit tests
+RUN git clone --depth 1 https://github.com/cgreen-devs/cgreen.git \
+    && cd cgreen \
+    && make \
+    && make test \
+    && make install \
+    && cd .. \
+    && rm -rf cgreen
+
+ENV LD_LIBRARY_PATH="/usr/local/lib"
```
Comment (bjoernricks) [inline .docker/testing/Dockerfile:81]: IMHO this should not be necessary but it doesn't harm.

Comment (y0urself) [reply .docker/testing/Dockerfile:81]: afaik it did not work without it, but I can try to remove it and run the CI again without it ...

**Review by bjoernricks — APPROVED** [review]

**Comment by y0urself** [comment]
addressing AP-1493

**Pull request merged**